.github/workflows/spectral-validation.yml

```diff
@@ -31,5 +31,5 @@ jobs:
         with:
           node-version: v20
       - run: npm install @stoplight/spectral-cli
-      - run: npx spectral lint --ruleset spectral/instantiation/validation-rules.yaml 'calm/samples/**/*(*.json|*.yaml)'
```

Comment:
@willosborne assume this is ok with your checks?

Reply:
Its worth mentioning that post this PR we do need to think about the validation rules we could put into flow specs

Reply:
So now we're only validating the one file? I think that's okay, if we only have one example they actually work on. 

These validations need some work anyway, we want to run the CALM CLI properly instead of just the spectral rules.